docs/setting-up.md

```diff
@@ -34,6 +34,7 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
 1. Set your `master` branch to track the main repo's `master` branch.
    ```sh
    git checkout master
+   git fetch upstream
```

Comment:
This should be on the previous step as the previous step is essentially setting up the remote itself. This step can then be verified with `git branch -r`.